membershipextras.php

```diff
@@ -177,7 +177,8 @@ function membershipextras_civicrm_pre($op, $objectName, $id, &$params) {
   }
 
   if ($objectName === 'Membership' && $op == 'edit') {
-    $membershipPreHook = new CRM_MembershipExtras_Hook_Pre_MembershipEdit($id, $params, $contributionID);
+    $paymentType = Civi::$statics[E::LONG_NAME]['paymentType'] ?? '';
+    $membershipPreHook = new CRM_MembershipExtras_Hook_Pre_MembershipEdit($id, $params, $contributionID, $paymentType);
     $membershipPreHook->preProcess();
   }
```

Comment:
Just removed this line at line 182
```diff
- unset(Civi::$statics[E::LONG_NAME]['paymentType']);
```
because if you created a pending memberships in one go from a priceset then this `if` condition will be true for each membership and we need to keep this `Civi::$statics[E::LONG_NAME]['paymentType']` for the other memberships